pkg/cache/ttl.go

```diff
@@ -163,6 +163,11 @@ func (c *ttlCache) doGC() {
 					}
 				}
 			}
+			if len(c.items) == 0 {
```

Comment:
How about `remove`?

Reply:
it doesn't matter, one gc interval will exit.

Reply:
good catch, I have added one atomic flag to check the goroutine exist.